### PR TITLE
Update classwork for day 1 to taxi data

### PR DESCRIPTION
**Pull request by hfrick**
Staying close to the frog data version, biggest change is probably in the use of the metrics since this is now a classification problem

### CODE REVIEW

File: classwork/02-classwork.qmd
Comment (hfrick) [inline classwork/02-classwork.qmd:1]: Let's keep this and #108 in sync :)

File: classwork/02-classwork.qmd [hunk 1]
```diff
@@ -77,7 +77,7 @@ Explore the `frog_train` data on your own!
 ```{r}
 set.seed(123)
 
-frog_split <- initial_split(tree_frogs, prop = 0.8, strata = latency)
-frog_train <- training(frog_split)
-frog_test <- testing(frog_split)
+taxi_split <- initial_split(taxi, prop = 0.8, strata = tip)
```
Comment (hfrick) [inline classwork/02-classwork.qmd:80]: tbd: stratify by distance or tip?

File: classwork/03-classwork.qmd [hunk 2]
```diff
@@ -101,16 +101,16 @@ This is how you'd fit a decision model:
...
 tree_wflow <- workflow() %>%
-  add_formula(latency ~ .) %>%
+  add_formula(tip ~ .) %>%
   add_model(tree_spec)
 
 tree_wflow
 ```
 
-Now use a similar approach to fit a linear model! Call it `lm_wflow`.
+Now use a similar approach to fit a different model!
```
Comment (hfrick) [inline classwork/03-classwork.qmd:113]: We could be prescriptive about the model here and ask them for a logistic regression.

File: classwork/04-classwork.qmd [hunk 3]
```diff
@@ -13,57 +13,57 @@ Setup from deck 3
...
 
 ## Metrics for model performance
 
 `metrics()` returns a standard set of metrics
 
 ```{r}
-augment(tree_fit, new_data = frog_test) %>%
-  metrics(latency, .pred)
+augment(tree_fit, new_data = taxi_test) %>%
+  metrics(tip, .pred_class, .pred_yes)
```
Comment (hfrick) [inline classwork/04-classwork.qmd:42]: show hard and soft class metrics or just one?

File: classwork/04-classwork.qmd [hunk 4]
```diff
@@ -165,27 +165,26 @@ If you want to analyze the assessment set (i.e. holdout) predictions, then you n
...
 
 ```{r}
 tree_preds %>% 
-  ggplot(aes(latency, .pred, color = id)) + 
-  geom_abline(lty = 2, col = "gray", size = 1.5) +
-  geom_point(alpha = 0.5) +
-  coord_obs_pred()
+  ggplot(aes(.pred_class, fill = id)) + 
+  geom_bar(alpha = 0.5) +
+  facet_grid(. ~ tip)
```
Comment (hfrick) [inline classwork/04-classwork.qmd:180]: Do we want this or a different plot?

File: classwork/02-classwork.qmd (hunk 1 above)
Comment (simonpcouch) [reply classwork/02-classwork.qmd:80]: you got it! addressed in #108.

File: classwork/03-classwork.qmd (hunk 2 above)
Comment (EmilHvitfeldt) [reply classwork/03-classwork.qmd:113]: I would be game for that

File: classwork/04-classwork.qmd (hunk 3 above)
Comment (EmilHvitfeldt) [reply classwork/04-classwork.qmd:42]: I think we need to show both eventually.

I think we should avoid `metrics()` altogether

Comment (hfrick) [reply classwork/04-classwork.qmd:42]: I'm game! Do what you think is good in the updates to deck 4 and then we'll update the classwork accordingly

File: classwork/03-classwork.qmd (hunk 2 above)
Comment (hfrick) [reply classwork/03-classwork.qmd]: edited in [c694e26](https://github.com/tidymodels/workshops/pull/109/commits/c694e261bc079940b36cc85f849d8df6ce60c367) ✔️

**Pull request merged**